### ISSUE TITLE
Parsing error with parametrized macro

### ISSUE THREAD

**Issue by zsoci** [AdRoll/rebar3_format#84]
```erlang
-define(EXCEPTION(C, R, Stacktrace), C:R:Stacktrace).
```

fails with 

```
===> Uncaught error in rebar_core. Run with DEBUG=1 to see stacktrace or consult rebar3.crashdump
===> Uncaught error: {error,<<"Formatting error: [\"syntax error before: \",\"?\"].Try running with DEBUG=1 for more information">>}
```

**Comment by elbrujohalcon** [comment]
Nice catch, @zsoci !! It's not the same but probably related to #22

**Comment by elbrujohalcon** [comment]
This comment applies here as well: https://github.com/AdRoll/rebar3_format/issues/22#issuecomment-619989513

**Comment by elbrujohalcon** [comment]
This was fixed with #207.
